src/main/webapp/timezone.jsp

```diff
@@ -19,8 +23,10 @@
         <script type="text/javascript" src="/js/lib/moment.min.js"></script>
         <script type="text/javascript" src="/js/lib/moment-timezone-with-data-2010-2020.min.js"></script>
         <script>
+            var d = new Date();
             moment.tz.names().forEach(function(timeZone) {
-                $('#momentjs').append(timeZone + '<br>');
+                var offset = moment.tz.zone(timeZone).offset(d) * -1;
+                $('#momentjs').append(timeZone + ' ' + offset + '<br>');
```

Comment:
Not really, but the said JSP was not strong enough as test before. While it did account for the list of timezone, it failed to account that each timezone was associated to the same offset, which is what was done here.